helpers/htmlAttributeObserverController.js

```diff
@@ -1,24 +1,25 @@
 const controllerCallbacks = new Map();
 
 const contextObserver = new MutationObserver(mutations => {
-	mutations.forEach(mutation => {
-		controllerCallbacks.forEach(callback => callback(mutation.attributeName));
-	});
+	controllerCallbacks.forEach(callback => callback(
+		mutations.map(mutation => mutation.attributeName)
+	));
 });
 
 export class HtmlAttributeObserverController {
 
-	constructor(host, attribute) {
-		this._host = host;
-		this._attribute = attribute;
+	constructor(host, ...attributes) {
```

Comment:
Do you think it is better to pass the attribute names on the `constructor`, or should they be passed as an arg to `hostConnected`?  i.e. could there be use-case where we do not know the attributes to observer until after the component is constructed?

Reply:
Hmm, that's a good point. My intent here was to simulate what an actual reactive controller would do (which, as I understand it, would handle the `hostConnected` and `hostDisconnected` under the hood). But I'm also fine changing this behaviour. Certainly right now the attributes we're watching are immutable per controller, but it doesn't have to be that way.

Reply:
If we're expecting this to eventually be a controller derived from something Lit provides, where those methods are called automatically, then I would stick with what you have.

Reply:
Fair. I'll double check to make sure I'm not misunderstanding the controller interface. If I'm not, I'll leave it as is.